scout/server/blueprints/institutes/templates/overview/phenomodel.html

```diff
@@ -172,9 +172,12 @@ <h6 class="text-muted ml-3 mt-3">Created: {{phenomodel.created.strftime('%Y-%m-%
     {% for subpanel_id, subpanel in phenomodel.subpanels.items()%}
       <div class="card">
         <div class="card-header">
-          <strong>{{subpanel.title}}</strong>&nbsp;&nbsp;<text class="text-muted">{{subpanel.subtitle}}</text>
+          <a href="#" onclick="toggleDiv('{{subpanel_id}}')">
```

Comment:
Panel toggle works. I noticed that all subpanels are expanded when you go to a panel page. Could we have it the other way around instead, with all subpanels hidden and shown only if you click on them?

Reply:
We fixed it